frontend/test/account-send.test.ts

```diff
@@ -781,3 +782,72 @@ describe('End to end account tests', () => {
     20000
   );
 });
+
+describe('clearHistory', () => {
+  beforeEach(async () => {
+    await localforage.clear();
+    window.logger = {
+      debug: jest.fn(),
+      info: jest.fn(),
+      warn: jest.fn(),
+      version: '',
+      _log: jest.fn(),
+      makeError: jest.fn(),
+      assert: jest.fn(),
+      assertArgument: jest.fn(),
+      checkNormalize: jest.fn(),
+      checkArgumentCount: jest.fn(),
+      checkNew: jest.fn(),
+      checkAbstract: jest.fn(),
+      checkSafeUint53: jest.fn(),
+      throwError: jest.fn() as never,
+      throwArgumentError: jest.fn() as never,
+    };
+  });
+
+  it.each([randomInt(2, 10), randomInt(2, 10), randomInt(2, 10), randomInt(2, 10)])(
```

Comment:
1. Should the lower bound start at zero here, to make sure the function behaves properly even when there's no data? (Should basically be a no-op in that case). Or at least start at 1
2. We should raise the upper bound too, 10 feels unnecessarily small

Reply:
Handled https://github.com/ScopeLift/umbra-protocol/pull/546/commits/19c3968baed36c9d321dea44931bdc86f8c2ccca and I changed to 0 in another commit

Reply:
I raised to 50. I tried 100, but it caused the tests timeout, and I already overrode the default timeout